### PR TITLE
Add seats to print view and userFrontend

### PR DESCRIPTION
**Pull request by ffactory-ofcl**

### CODE REVIEW

File: src/serverMain/kotlin/com/studo/campusqr/endpoints/QrCodes.kt [hunk 1]
```diff
@@ -46,7 +46,10 @@ suspend fun AuthenticatedApplicationCall.viewSingleQrCode() {
             ).get(this@viewSingleQrCode)
           }
         }
-        renderLocation(location, configs)
+        div {
+          id = "allcodes"
```
Comment (vonox7) [inline src/serverMain/kotlin/com/studo/campusqr/endpoints/QrCodes.kt:50]: here and everywhere: allcodes is not a single word. I think you mean allCodes ;)

File: src/serverMain/kotlin/com/studo/campusqr/endpoints/QrCodes.kt [hunk 2]
```diff
@@ -104,29 +114,38 @@ suspend fun AuthenticatedApplicationCall.viewAllQrCodes() {
...
+          +configs.getValue("scanSubtext2")
+        }
       }
     }
+    div("break") {}
+  }
+
+  if (location.seatCount != null) {
+    for (seat in 1..location.seatCount) {
+      val paddedSeat = seat.toString().padStart(location.seatCount.toString().length, '0')
```
Comment (vonox7) [inline src/serverMain/kotlin/com/studo/campusqr/endpoints/QrCodes.kt:145]: nice logic!

File: src/serverMain/kotlin/com/studo/campusqr/endpoints/UserFrontend.kt [hunk 3]
```diff
@@ -14,17 +12,15 @@ import java.util.*
 
 
 suspend fun ApplicationCall.userFrontend() {
-  val locationId = parameters["l"]
-  val locationName = locationId?.let { id -> getLocationOrNull(id)?.name }
-  val language = this.language
+  val fullLocationId = parameters["l"]
+  var seatNr: Int? = null
```
Comment (vonox7) [inline src/serverMain/kotlin/com/studo/campusqr/endpoints/UserFrontend.kt:16]: you call it here `seatNr`, on other occurences it is called just `seat`. We should stick to one naming if it is the same.

File: src/serverMain/kotlin/com/studo/campusqr/endpoints/UserFrontend.kt [hunk 4]
```diff
@@ -14,17 +12,15 @@ import java.util.*
...
+  val locationId = if (fullLocationId?.contains("-") == true) {
+    seatNr = fullLocationId.substringAfterLast("-", "").trimStart('0').toIntOrNull()
```
Comment (vonox7) [inline src/serverMain/kotlin/com/studo/campusqr/endpoints/UserFrontend.kt:18]: Pls check if we scan a paper with "#001" we also show in the user frontend "#001" and not "#1"

**Review by vonox7 — CHANGES_REQUESTED** [review]

File: src/serverMain/kotlin/com/studo/campusqr/endpoints/UserFrontend.kt (hunk 4 above)
Comment (vonox7) [reply src/serverMain/kotlin/com/studo/campusqr/endpoints/UserFrontend.kt:18]: ok, github reformatted my message :D

Comment (ffactory-ofcl) [reply src/serverMain/kotlin/com/studo/campusqr/endpoints/UserFrontend.kt:18]: > image
lol

File: src/serverMain/kotlin/com/studo/campusqr/endpoints/QrCodes.kt [hunk 5]
```diff
@@ -75,47 +55,52 @@ suspend fun AuthenticatedApplicationCall.viewAllQrCodes() {
...
-        script {
-          src = "/static/viewQR/generate.js"
-        }
+        viewQrCodes(locations, configs, language)
       }
     }
   }
 }
 
+fun BODY.viewQrCodes(locations: List<ClientLocation>, configs: Map<String, String>, language: String) {
```
Comment (vonox7) [inline src/serverMain/kotlin/com/studo/campusqr/endpoints/QrCodes.kt:69]: view? or render? we use render everywhere else for this type

**Review by vonox7 — APPROVED** [review]

**Pull request merged**